### ISSUE TITLE
Definition of escape for escapedHTMLString()

### ISSUE THREAD

**Issue by vmarta** [mobify/mobifyjs#133]
Why is escapedHTMLString() called as it is now? Shouldn't it be unescapedHTMLString() perhaps?

In the doc's [Quick Start](http://www.mobify.com/mobifyjs/v2/docs/) page, the main.js sample has this comment:

> // To escape content, we prepend resources with "x-",
> // so to change the src, you must set x-src.

In this case, the word 'escape' means _adding_ the prefix.

However, that meaning is contradicted with what [escapedHTMLString()](http://www.mobify.com/mobifyjs/v2/docs/capturing/#escapedhtmlstring) does:

> Returns a string representation of the captured document, but with all resources enabled (prefix removed).

What's the correct definition of 'escape'? Thanks.

**Comment by jansepar** [comment]
You're totally right @vmarta, it should be `enabledHTMLString`! Sorry for taking a bigillian years to get to this :)
